### PR TITLE
Kdreval dev

### PR DESCRIPTION
**Pull request by Kdreval**
Here, I have:
1. Resolved R4-specific error related to the changed use of read_tsv mentioned in issue #58 
2. Added support for consistent hotspot annotation in LymphGen-consistent way for NOTCH1, NOTCH2, CD79B, and EZH2 mentioned in issue #57

### CODE REVIEW

**Review by mattssca — COMMENTED** [review]
Nice updates to the scripts, I am assuming you've tested the functions involved (on the different versions of R)? 

Just a thought, would there be any benefit from generating two bed files (one for hg19 and one for hg38) for the genes of interest, instead of hardcoding the coordinates inside the functions? In that way, adding more genes in the future would not require any updates to the code itself?

Since my edits have not been merged yet, I understand it can be tricky to grasp a syntax-standard set by me in my revisions. That being said, I could easily update this code to match the standard once approved.

File: R/preprocessing_io.R [hunk 1]
```diff
@@ -808,11 +808,15 @@ liftover_bedpe = function(bedpe_file,bedpe_df,target_build="grch37",col_names,co
     original_bedpe = as.data.frame(original_bedpe)
     original_bedpe=original_bedpe %>% mutate_if(is.numeric, as.integer)
     #print(head(original_bedpe))
-    if(!grepl("chr",original_bedpe$CHROM_A)){
-      #add chr prefix
-      original_bedpe = original_bedpe %>% mutate(CHROM_A = paste0("chr",CHROM_A)) %>%
-      mutate(CHROM_B = paste0("chr",CHROM_B))
-    }
+    original_bedpe =
```
[the remark this answers is not in the document]
Comment (Kdreval) [reply R/preprocessing_io.R]: I did reformatted it, thanks for the example!

**Comment by Kdreval** [comment]
Thanks @mattssca  for the review! I did address the comments and updated the syntax/implementation. This is now ready for re-review, and I confirmed that it gives consistent outputs. One new thing that was pushed to this PR is a change in `prettyOncoplot` to allow for plotting genes with 0 mutations.

**Review by mattssca — APPROVED** [review]
Thanks for making the updates Kostia, I think it's a better approach to use bed-style files instead of hard-coding regions. It will allow for an easy update if additional regions need to be added in the future. Glad you agreed.

**Pull request merged**